### DISCUSSION TITLE
How to add supported hardware?

### DISCUSSION THREAD

**Discussion by Ashley-corey** [opensteno/plover#1256] [Q&A and Support] [answered]
I have found a new piece of hard ware that is supported, if any more are found they could be added in the discussion to possibly be added to the list.

the keyboard I use is the Marvo KG901, a relatively cheap keyboard going four around $45 AUD, although the price changes from time to time.

**Comment by user202729** [comment, marked as answer]
I think you're supposed to edit the wiki page directly.

... perhaps a list of "keyboards that are tested and does **not** work" is more useful.

**Reply by Ashley-corey** [reply]
I don't know how to edit the wiki directly, thanks for the suggestion though. I will try to find out how to do so and do that

**Reply by user202729** [reply]
You should be able to edit the wiki...

![image](https://user-images.githubusercontent.com/25191436/114804018-bd4fa480-9dca-11eb-82db-f87b2b3b6fcb.png)

**Reply by Ashley-corey** [reply]
I figured it out thank you, somehow I managed to miss the edit button.
